bank-accounts.rb

```diff
@@ -6,13 +6,15 @@
 module Bank
 
   class Account
+    @@min_balance = 0
+
     attr_reader :balance, :owner, :open_date, :ident
 
-    def initialize(ident, balance, open_date)
+    def initialize(ident, open_date, balance)
       @ident = ident
-      @balance = balance
-      raise ArgumentError.new("Your intial balance can't be negative") if balance < 0
       @open_date = DateTime.strptime(open_date, '%Y-%m-%d %H:%M:%S %z')
+      @balance = balance
+      raise ArgumentError.new("Insufficient initial balance") if balance < @@min_balance
     end
 
     def self.all
```

Comment:
Nice job using the account and owner methods to drive your ultimate list functionality as well